app/components/post_body/post_body.js

```diff
@@ -407,21 +407,19 @@ export default class PostBody extends PureComponent {
         return (
             <View style={style.messageContainerWithReplyBar}>
                 <View style={replyBarStyle}/>
-                <View style={[style.row]}>
```

Comment:
we can remove style `row` here as it is not needed anymore

Reply:
done